lua/lvim/core/telescope/custom-finders.lua

```diff
@@ -49,11 +49,11 @@ function M.view_lunarvim_changelog()
     finder = finders.new_oneshot_job(
       vim.tbl_flatten {
         "git",
+        "--git-dir=" .. utils.join_paths(get_runtime_dir(), "lvim", ".git"),
+        "--work-tree=" .. utils.join_paths(get_runtime_dir(), "lvim"),
         "log",
         "--pretty=oneline",
         "--abbrev-commit",
-        "--",
-        ".",
       },
       opts
```

Comment:
```suggestion
            vim.tbl_deep_extend("force", opts, {cwd = get_lvim_base_dir()})
```
You can also add the cwd to line 43, but github doesn't let me suggest that far back 😄 
```lua
local opts = { cwd = get_lvim_base_dir() }
```
Similar to this: https://github.com/LunarVim/LunarVim/blob/87eb204735fcd495199c462d578ce0eb1904e46e/lua/lvim/bootstrap.lua#L176

Reply:
This worked like a charm! Thanks 🚀